Lib/platform.py

```diff
@@ -186,6 +186,10 @@ def libc_ver(executable=None, lib='', version='', chunksize=16384):
 
         executable = sys.executable
 
+        if not executable:
+            # sys.executable is not set.
+            return lib, version
```

Comment:
I suggest to move it at the beginning of the function:

```
if not executable and not sys.executable:
    # sys.executable is not set.
    return lib, version
```

The comment may mention WASI.

Reply:
That would by-pass the ``os.confstr('CS_GNU_LIBC_VERSION')`` lookup. Emscripten has a working confstr() implementation. WASI's confstr() API is not implemented **yet**.

Reply:
> That would by-pass the os.confstr('CS_GNU_LIBC_VERSION') lookup.

Ah right.